lms/djangoapps/discussion_api/api.py

```diff
@@ -145,6 +146,7 @@ def get_course(request, course_key):
             for blackout in course.get_discussion_blackout_datetimes()
         ],
         "thread_list_url": get_thread_list_url(request, course_key, topic_id_list=[]),
+        "following_thread_list_url": get_thread_list_url(request, course_key, following=True),
```

Comment:
nitpick: it's a little confusing that you pass `topic_id_list=[]` in the previous call and omit it entirely in this one...is there a reason for the difference?